fullscreen-api/main.js

```diff
@@ -0,0 +1,28 @@
+window.addEventListener("load", startup, false);
+
+function startup() {
+  // Get the reference to video
+  const video = document.getElementById("video");
+
+  // On pressing ENTER call toggleFullScreen method
+  document.addEventListener("keypress", function(e) {
+    if (e.keyCode === 13) {
```

Comment:
AFAIK, `keyCode` is deprecated, and `key` is more the best practice these days. Update it to use `key` instead?

Reply:
Yes this makes sense. I was trying to stay close to the example in MDN docs which used `keyCode`. I'll go ahead and update it as well I think.